tests/lib/collectors/bigquery.py

```diff
@@ -26,11 +28,27 @@ def testQuery(self, mock_bq):
     mock_bq().query().to_dataframe().to_json.return_value = "{'foo':1}"
     test_state = state.DFTimewolfState(config.Config)
     bq_collector = bigquery.BigQueryCollector(test_state)
-    bq_collector.SetUp('test_project', 'test_query', 'test_description')
+    bq_collector.SetUp('test_project', 'test_query', 'test_description', False)
```

Comment:
since False means the state should contain a File container, perhaps we could check that too?